README.md

```diff
@@ -1,23 +1,22 @@
-# apm - Atom Package Manager
+# ppm - Pulsar Package Manager
 
-![Build Status](https://github.com/atom-ide-community/apm/workflows/CI/badge.svg)
-[![Dependency Status](https://david-dm.org/atom/apm.svg)](https://david-dm.org/atom/apm)
+Discover and install Pulsar packages powered by [pulsar-edit.dev](https://web.pulsar-edit.dev).
```

Comment:
Yeah this has been my general strategy when dealing with the package site, I refer to the main website but link to the `web.pulsar-edit.dev` address so that it brings you to the actual package site, much in the same way that Atom would have the link displayed as `atom.io` going to `atom.io/packages`.